app/models/account/encrypted_data.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class Account::EncryptedData
```

Comment:
how about:

cleartext_data, data, iv

#encrypt(encryption_key), #decrypt(encryption_key)

#to_json implement your-self (returns the correct attrs, raises exception if #encrypt was not called -> data is empty)